pages/people/[id]/warning-notes/[warningNoteId]/[[...stepId]].tsx

```diff
@@ -18,21 +19,17 @@ const ReviewWarningNote = (): React.ReactElement => {
   const confirmation = asPath.includes('confirmation');
   const { user } = useAuth() as { user: User };
   const onFormSubmit = useCallback(
-    (person: Resident) => async ({
-      form_name,
-      ...formData
-    }: Record<string, unknown>) => {
-      console.log({
-        personId: person.id,
-        firstName: person.firstName,
-        lastName: person.lastName,
-        contextFlag: person.contextFlag,
-        dateOfBirth: person.dateOfBirth,
-        workerEmail: user.email,
-        formNameOverall:
-          person.contextFlag === 'A' ? 'ASC_case_note' : 'CFS_case_note',
-        formName: form_name,
-        caseFormData: JSON.stringify(formData),
+    (person: Resident) => async ({ ...formData }: Record<string, unknown>) => {
+      updateWarningNote(warningNoteId, {
+        warningNoteId,
+        reviewedBy: user.email,
+        endedBy: formData.reviewDecision === 'No' ? user.email : undefined,
+        endedDate:
+          formData.reviewDecision === 'No'
+            ? new Date().toISOString().substr(0, 10)
```

Comment:
Does it make more sense to pass a full ISO-8601 date here (without `.substr()`) or is this just the way the back end works?

Reply:
Yep, for the BE should be the same